config.js

```diff
@@ -723,7 +723,7 @@ config.TIERING_TTL_MS = 30 * 60 * 1000; // 30 minutes
 // AWS SDK VERSION     //
 /////////////////////////
 
-config.AWS_SDK_VERSION_3_DISABLED = true;
+config.AWS_SDK_VERSION_3_DISABLED = false;
```

Comment:
It's not too late, I can refactor it (in this PR or in a separate PR it doesn't matter).
I'll share my thoughts so we will find a better suggestion.
I use the suffix DISABLED to indicate that it's a boolean, the suggestion to remove it doesn't indicate what kind of value to expect.
The main use of this variable is in case the new AWS SDK V3 would create a bug and we want to switch back to AWS SDK V2 fast - the main use of it is here:
https://github.com/noobaa/noobaa-core/blob/06d01f2e4c1ac1cd14368f82e550946ec2e5c23b/src/sdk/noobaa_s3_client/noobaa_s3_client.js#L28-L33
Changing it to ENABLED can be a solution for the double negative you're raising, but it would create a negative condition in the function above (`return !config.AWS_SDK_VERSION_3_ENABLED`...).
WDYT?

Reply:
If it is confusing, then it should be changed.
I changed it to `AWS_SDK_VERSION_3_ENABLED`.